http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitor.scala

```diff
@@ -154,11 +169,14 @@ class LeoMetricsMonitor[F[_]](config: LeoMetricsMonitorConfig,
                   AppCreationException(s"Pet not found for user ${app.auditInfo.creator}", Some(ctx.traceId))
                 )
                 authHeader = Authorization(Credentials.Token(AuthScheme.Bearer, token))
-                relayPath = Uri.unsafeFromString(baseUri.asString) / app.appName.value
+                relayPath = Uri
+                  .unsafeFromString(baseUri.asString) / s"${app.appName.value}-${app.workspaceId.map(_.value.toString).getOrElse("")}"
                 isUp <- serviceName match {
                   case ServiceName("cbas")     => cbasDAO.getStatus(relayPath, authHeader).handleError(_ => false)
                   case ServiceName("cbas-ui")  => cbasUiDAO.getStatus(relayPath, authHeader).handleError(_ => false)
                   case ServiceName("cromwell") => cromwellDAO.getStatus(relayPath, authHeader).handleError(_ => false)
+                  case ServiceName("wds")      => wdsDAO.getStatus(relayPath, authHeader).handleError(_ => false)
```

Comment:
I noticed that WDS and HailBatch services were missing, causing them not to be counted currently.

Reply:
Yup, the GCP apps are all behind `HttpAppDAO`.